www/layout.html

```diff
@@ -39,24 +42,37 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarResponsive">
           <ul class="navbar-nav ml-auto">
+            {% if page_name == 'manage': %}
```

Comment:
I think we can save some duplicated lines here. The only difference the `<span>`. So consider writing the if-else inside the `<li>` something like:
```html
<li class="nav-item active">
             <a class="nav-link" href="/manage">Manage</a>
{% if page_name == 'manage': %}
             <span class="sr-only">(current)</span>
{% endif %}
</li>
```
This way we don't need the else statement ;)

Reply:
This is a tricky one because it's the <span> and the "active" class. I might just remove the span since it's for crawlers and readers and makes no difference to our front end, but still, it's the class that makes it tricky. I didn't want to get overly fancy with a solution for time reasons, but if needed -- we can go back to this.

Reply:
Oh, so if it is only the class we can do `<li class="nav-item {%if page_name == 'manage' %}active{% endif %}">` 
Maybe a [filter](http://jinja.pocoo.org/docs/2.9/templates/#equalto) could help?

Reply:
Yeah that's what i ended up doing (first suggestion, not the filter, but great article for other things!). I removed the span entirely and just left the active class as variable in a condensed line.. you can see it in the merge now ;)